crates/forge/src/cheatcodes_hint_processor.rs

```diff
@@ -437,8 +468,16 @@ fn execute_syscall(
     buffer.write(Felt252::from(exit_code)).unwrap();
 
     buffer.write_arr(result.iter()).unwrap();
+}
 
-    Ok(())
+fn execute_keccak(
+    original_cairo_hint_processor: &mut OriginalCairoHintProcessor,
+    vm: &mut VirtualMachine,
+    exec_scopes: &mut ExecutionScopes,
+    hint_data: &Box<dyn Any>,
+    constants: &HashMap<String, Felt252>,
+) -> Result<(), HintError> {
+    original_cairo_hint_processor.execute_hint(vm, exec_scopes, hint_data, constants)
 }
```

Comment:
Do we need this function 🤔 It would be more readable if we directly called `original_cairo_hint_processor.execute_hint` from pattern match. You would know that it is offloaded.

Reply:
Otherwise if it stays we could probably mark this `#inline`?